packages/cli/src/commands/projects.ts

```diff
@@ -54,6 +69,7 @@ const main = async (client: Client) => {
     argv = getArgs(client.argv.slice(2), {
       '--next': Number,
       '-N': '--next',
+      '--yes': Boolean,
```

Comment:
This should be `--confirm` right? Like `vc dev --confirm` and `vc link --confirm`?